### PR TITLE
Add effect for Crimson Shroud AC bonus

### PR DESCRIPTION
**Pull request by LebombJames**

### CODE REVIEW

**Comment by stwlam** [comment]
Eh, not worth making an interact action for it

**Comment by LebombJames** [comment]
I've got a version granting an action with the AC bonus as a self-applied effect locally. Imo it works a bit smoother. Lmk if you want that instead

File: packs/feat-effects/effect-crimson-shroud-ac.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+    "_id": "gvE0iQ7h6263Mkhi",
+    "img": "icons/magic/defensive/shield-barrier-glowing-triangle-red.webp",
+    "name": "Effect: Crimson Shroud (AC)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Crimson Shroud].</p>"
```
Comment (TikaelSol) [inline packs/feat-effects/effect-crimson-shroud-ac.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Crimson Shroud].</p>/n<p>You gain a +1 circumstance bonus to your AC.</p>"
```

File: packs/actions/swirl-crimson-shroud-interact.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            "title": "Pathfinder Lost Omens: World Guide"
+        },
+        "rules": [],
+        "selfEffect": {
+            "name": "Effect: Crimson Shroud (AC)",
+            "uuid": "Compendium.pf2e.feat-effects.Item.Effect: Crimson Shroud (AC)"
+        },
+        "traits": {
+            "value": [
+                "general",
```
Comment (TikaelSol) [inline packs/actions/swirl-crimson-shroud-interact.json:28]: General?

File: packs/actions/swirl-crimson-shroud-interact.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+    "_id": "Gu0QsFvWoR9EP5SE",
+    "img": "icons/magic/defensive/shield-barrier-glowing-triangle-red.webp",
+    "name": "Swirl Crimson Shroud (Interact)",
```
Comment (TikaelSol) [inline packs/actions/swirl-crimson-shroud-interact.json:4]: I think we can lose the parenthetical

**Pull request merged**